### PR TITLE
Deploy to Production

### PR DESCRIPTION
**Pull request by cesarvarela**
## Before merge instructions

To get tests passing on production, the test user needs to have his `first_name` and `last_name` set.

- look for the `userId` of the test user in the admin table
- manually update his associated user document in the `users` collection like this:
image

(Tests expect `{first_name: 'Test', last_name: 'User'}`)

### CODE REVIEW

**Comment by cesarvarela** [comment]
@smcgregor ready for review, special instructions before merging in the pr description.

**Comment by smcgregor** [comment]
@cesarvarela How do I `look for the userId of the test user in the admin table`? How is this user identified?

**Comment by cesarvarela** [comment]
@smcgregor it's the same user that's set on Netlify:

image

**Comment by smcgregor** [comment]
I did the following (modified where obvious):

1. Get the admin username from Netlify's environment variables, which is in the `E2E_ADMIN_USERNAME` field. For example `prod@incidentdatabase.ai`

2. Open MongoDB Atlas's App Services for the app you want to test against, click on `App Users` and find the admin user with the matching email address (`prod@incidentdatabase.ai`). Copy its Id. For example: `123456789ddd11111111111a`

2. Go back to `Data Services` and select the collection that has the user table under `customData`. In the users table, filter to the Id you copied. For example, `{userId:"123456789ddd11111111111a"}`

3. On the single matching document, add the following fields: `{first_name: "Test", last_name:"User"}`.

Should be good to go, but I will update here if I run into an issue at merge time.

File: site/gatsby-site/src/templates/landingPage.js [hunk 1]
```diff
@@ -51,12 +52,30 @@ const LandingPage = (props) => {
...
+
   return (
     // Tailwind has max-w-6xl but no plain w-6xl... 72rem = 6xl
     <Layout {...props} className="max-w-full 2xl:w-[72rem]">
       <AiidHelmet {...{ metaTitle, metaDescription, path: props.location.pathname, metaImage }}>
         <title>{title}</title>
         <meta property="og:type" content="website" />
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{ __html: JSON.stringify(ldJSON) }}
```
Comment (smcgregor) [inline site/gatsby-site/src/templates/landingPage.js:77]: Is there another way?

**Pull request merged**